.ci/docker/buildtools-windows/Dockerfile

```diff
@@ -6,21 +6,26 @@ FROM mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-1909
 # Restore the default Windows shell for correct batch processing.
 SHELL ["cmd", "/S", "/C"]
 
-# Download the Build Tools bootstrapper.
-ADD https://aka.ms/vs/16/release/vs_buildtools.exe C:\TEMP\vs_buildtools.exe
-
-# Install Build Tools excluding workloads and components with known issues.
-RUN C:\TEMP\vs_buildtools.exe --quiet --wait --norestart --nocache `
-    --installPath C:\BuildTools `
-    --all `
-    --remove Microsoft.VisualStudio.Component.Windows10SDK.10240 `
-    --remove Microsoft.VisualStudio.Component.Windows10SDK.10586 `
-    --remove Microsoft.VisualStudio.Component.Windows10SDK.14393 `
-    --remove Microsoft.VisualStudio.Component.Windows81SDK `
- || IF "%ERRORLEVEL%"=="3010" EXIT 0
-
-# Start developer command prompt with any other commands specified.
-ENTRYPOINT C:\BuildTools\Common7\Tools\VsDevCmd.bat &&
+# Download NuGet v5.4.0 
+ADD https://dist.nuget.org/win-x86-commandline/v5.4.0/nuget.exe C:\nuget\nuget.exe
+
+# Add NuGet to Beginning of Path (We want this version of Nuget before any other versions that may be in the path)
+RUN @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "[Environment]::SetEnvironmentVariable(\"Path\", \"C:\nuget;\" + $env:Path, [EnvironmentVariableTarget]::Machine)"
+
+#This route takes 20-30 minutes
+# We need a place to download this from adding it to Github and copying locally is a potential option 
```

Comment:
I'm not sure how to download that specific version, but as far as I can download another version using the below curl command:

```bash
curl -I https://download.visualstudio.microsoft.com/download/pr/b79d5d47-f792-4eec-adcf-2da2dfa3e3db/4a244b3dd7c258763614d6e4eb914d42f47917e7cc6a70b509e9d9029aceddf2/vs_Enterprise.exe
HTTP/2 200 
accept-ranges: bytes
access-control-allow-origin: https://www.visualstudio.com
cache-control: public, max-age=259200
content-disposition: attachment; filename=vs_Enterprise.exe
content-type: application/octet-stream
date: Wed, 15 Jan 2020 09:28:56 GMT
etag: "4460df34bc7d51:0"
last-modified: Fri, 10 Jan 2020 00:21:49 GMT
server: ECAcc (lhb/6320)
x-cache: HIT
x-powered-by: ASP.NET
content-length: 1382880

```

That particular URL is the one I got when debugging the link to the vs_enterprise.exe file in https://docs.microsoft.com/en-us/visualstudio/install/create-a-network-installation-of-visual-studio?view=vs-2019

![image](https://user-images.githubusercontent.com/2871786/72422229-f4737800-3779-11ea-8474-f42308d3ad91.png)


Besides, it seems the chocolatey approach does use that particular URL to download the vs_enterprise:

- https://chocolatey.org/packages/visualstudio2019enterprise#files
- Click on the `Show` button for the `tools\ChocolateyInstall.ps1`

https://chocolatey.org/packages/visualstudio2019enterprise/16.4.0.0#files seems to be the one used for the current chocolatey approach

What do you think to use the URL from that particular `tools\ChocolateyInstall.ps1` script?